src/method/method_scan.cpp

```diff
@@ -151,11 +151,11 @@ namespace cubscan
 	      DB_VALUE *dbval_p = (DB_VALUE *) db_private_alloc (m_thread_p, sizeof (DB_VALUE));
 	      db_make_null (dbval_p);
```

Comment:
necessary to check "dbval_p is null"

Reply:
I've fixed to check dbval_p is null and make to occur an error